From 47f6a3bcf7e72c467ca6a2cd633a5c3585a218d5 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 2 Oct 2020 06:44:53 +0000
Subject: [PATCH] add dependency on openh264:encoder

webcodecs uses openh264 encoder, but dependency is missing. With
unbundled openh264 library build fails, because include path is
incomplete.
---
 third_party/blink/renderer/modules/webcodecs/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/blink/renderer/modules/webcodecs/BUILD.gn b/third_party/blink/renderer/modules/webcodecs/BUILD.gn
index 99d0e47..7c4d188 100644
--- a/third_party/blink/renderer/modules/webcodecs/BUILD.gn
+++ b/third_party/blink/renderer/modules/webcodecs/BUILD.gn
@@ -54,6 +54,7 @@ blink_modules_sources("webcodecs") {
     "//media/mojo/clients",
     "//media/mojo/mojom",
     "//third_party/libyuv:libyuv",
+    "//third_party/openh264:encoder",
   ]
   public_deps = [
     "//third_party/blink/renderer/modules/mediastream:mediastream",
-- 
2.26.2

